eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -187,11 +187,8 @@ private static void run(final List<String> opts) throws Exception {
             phi.attr("Δ").put(new Data.Value<>(arg));
             app.attr(0).put(phi);
         }
-        if (!new Dataized(app).take(Boolean.class)) {
-            throw new IllegalStateException(
-                "Runtime dataization failure"
-            );
-        }
+        final Object result = new Dataized(app).take();
+        Main.LOGGER.info(String.format("The result of %s dataization is %s", app, result));
```

Comment:
@yasamprom this will look weird in the console. It's better to make a more meaningful message, at least mentioning the type of data returned.